client/src/services/api.service.js

```diff
@@ -0,0 +1,11 @@
+import axios from 'axios';
+
+/**
+ * Instantiate axios instance
+ */
+
+const api = axios.create({
+    baseURL: '/api'
```

Comment:
please could you format the new files with 2-space indentation?

Reply:
Yeah of course. I think my VSCode defaulted to 4 for some reason.